app/components/support_interface/course_choices_table_component.html.erb

```diff
@@ -11,7 +11,13 @@
     <% course_rows.each do |course_option| %>
       <tr class='govuk-table__row'>
         <td class='govuk-table__cell'><%= course_option.id %></td>
-        <td class='govuk-table__cell'><%= govuk_link_to(course_option.course.name_and_code, support_interface_course_path(course_option.course)) %> - <%= course_option.study_mode.humanize %> at <%= course_option.site.name %></td>
+        <td class='govuk-table__cell'>
+          <% if course_option.invalidated_by_find? %>
+            <%= render TagComponent.new(text: 'Site deleted', type: 'red') %>
```

Comment:
@malcolmbaig should this read "Course no longer offered at this site"?

Reply:
Yes, with the concept of 'sites being deleted on Find' handled separately pending confirmation that this is actually a thing.